lib/iris/tests/unit/coords/__init__.py

```diff
@@ -14,7 +14,115 @@
 #
 # You should have received a copy of the GNU Lesser General Public License
 # along with Iris.  If not, see <http://www.gnu.org/licenses/>.
-"""Unit tests for the :mod:`iris.coords` module."""
+"""
+Unit tests for the :mod:`iris.coords` module.
 
+Provides test methods and classes common to test_AuxCoord and test_DimCoord.
```

Comment:
For completeness, you could include sphinx directives around `test_AuxCoord` and `test_DimCoord`.